palette/moreland/convert.go

```diff
@@ -0,0 +1,232 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package moreland
+
+import (
+	"image/color"
+	"math"
+)
+
+// rgb repesents a physically linear RGB color.
```

Comment:
I think sRGB is almost perceptually linear, but I think what we're going for here is linear in terms of light energy or whatever rather than perception, because that's what we want to be using for the color math.